templates/extras/calc.md

```diff
@@ -121,7 +109,7 @@ is `=`. Lean knows
 #check trans_rel_left -- {α : Sort u} {a b c : α} (r : α → α → Prop) (h₁ : r a b) (h₂ : b = c) : r a c
 ```
 
-and (Kevin believes) uses them if one of the operators is an equality operator. If however neither
+and uses them if one of the operators is an equality operator. If however neither
```

Comment:
Note that I am not sure the sentence is correct, but anyway in practice it works like it were.

Reply:
Yes, these are [instTransEq](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#instTransEq) and [instTransEq_1](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#instTransEq_1) (not sure if they have the highest priority, but there shouldn't be any conflicting instances).